### ISSUE TITLE
Get stats of training/test datasets

### ISSUE THREAD

**Issue by lmove** [akago/breaking_updates_rl#3]
## Description
It would be good to get a better understanding of the training/test dataset distributions (e.g., BC types, samples, projects). It won't necessarily be used in the article but can give a better understanding of the data.

## Solution
Consider the following steps to provide a solution:
- [ ] Extract project information per file.
- [ ] Have a post-processing analysis stage showing:
   - [ ] Test set distribution in terms of BC type (single-label vs multi-label)
   - [ ] Test set distribution in terms of project
   - [ ] Training set distribution in terms of BC type (single-label vs multi-label)
   - [ ] Training set distribution in terms of project

## Open Questions
- How to visualise this information for the multiple training tests?
